WordPress/Tests/WP/ClassNameCaseUnitTest.inc

```diff
@@ -65,3 +65,21 @@ class NotYetDetected {
 	public function paramTypeDeclaration( wp_role $role ) {}
 	public function returnTypeDeclaration() : WP_TERM {}
 }
+
+/*
+ * Safeguard correct handling of namespaced class references (the namespace types not handled below are already handled
+ * in other tests above).
+ */
```

Comment:
I think it would make sense to include some tests with `implements` here as the "class name" lists in the sniff also include interface names.